string modality
string second_information
---
bool aruco_found
int32 id_aruco
float32 x
float32 y
float32 z
float32[] vector
int32[] dim#dim[0]=rows; dim[1]=columns
int32 moreTargets
float32 finger_joint
bool[] all_aruco_found
